content/en/docs/concepts/workloads/autoscaling.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Autoscaling Workloads
+description: >-
+  With autoscaling, you can automatically update your workloads in one way or another. This allows your cluster to react to changes in resource demand more elastically and efficiently.
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+In Kubernetes, you can _scale_ a workload depending on the current demand of resources.
+This allows your cluster to react to changes in resource demand more elastically and efficiently.
+
+There are manual and automatic ways to scale your workloads, depending on your use case.
+
+<!-- body -->
+
+## Scaling workloads manually
+
+Kubernetes supports _manual scaling_ of workloads, either by changing the number of
+{{< glossary_tooltip text="Replicas" term_id="replica">}} of an object that manages a set of
+{{< glossary_tooltip text="Pods" term_id="pod" >}} (for example a {{< glossary_tooltip text="Deployment" term_id="deployment" >}}),
+or by adjusting the provided resources of each Replica (for example CPU or memory).
+
+### Scaling the number of Replicas of a workload
+
+You can use the `kubectl scale` command to increase or decrease the number of Replicas of a workload:
+
+```shell
+kubectl scale deployment <deployment-name> --replicas=<desired-replicas>
+```
+
+See also this [example of scaling a Deployment](/docs/concepts/workloads/controllers/deployment/#scaling-a-deployment) in the `Deployment` documentation.
+
+### Resizing workloads in-place
+
+Instead of scaling the number of Replicas of a workload, you can also adjust the provided resources
+of each Replica, in-place. You do this by patching the entries in one or both of the following
+fields of the `Pod` or [PodTemplate](/docs/concepts/workloads/pods/#pod-templates) you want to resize:
```

Comment:
```suggestion
Instead of scaling the number of replicas of a workload, you can also adjust the provided resources
for a particular pod or set of pods, in-place. You do this by patching the entries in one or both of the following
fields of the `Pod` or [pod template](/docs/concepts/workloads/pods/#pod-templates) you want to resize:
```